custom/icds_reports/static/js/directives/program-summary/program-summary.directive.js

```diff
@@ -133,12 +133,27 @@ function ProgramSummaryController($scope, $http, $log, $routeParams, $location,
 
     vm.getDataForStep(vm.step);
     vm.currentStepMeta = vm.steps[vm.step];
+
+    // mobile only, update if filters are visible over the program summary
+    vm.filtersOpen = false;
+    $scope.$on('openFilterMenu', function () {
```

Comment:
apologies for lack of my context. How do we ensure these only run on mobile?

Reply:
Well the event only gets raised via interaction with the mobile UI. So it's not exactly a complete assurance but that's why we wouldn't expect it to be relevant on web.